app/views/shared/_alert.html.erb

```diff
@@ -0,0 +1,12 @@
+<%# locals: (type: "error", content: "") -%>
+<% color = type == "error" ? "red" : "yellow" %>
+<%= content_tag :div,
+  class: "flex justify-between bg-#{color}-50 rounded-xl p-3",
```

Comment:
Since Tailwind does static analysis of the classes, `bg-#{color}-50` won't always work properly:

https://tailwindcss.com/docs/content-configuration#dynamic-class-names

Reply:
Great point, I always forget about this 🙇